source/mainnet/smart-contracts/guides/build-schema.rst

```diff
@@ -180,3 +180,36 @@ where ``FILE`` is a path of the file to create:
 .. code-block:: console
 
    $cargo concordium build --schema-out="/some/path/schema.bin"
+
+If using ``cargo concordium`` version 2.6.0 or newer then the schema can be
+output in JSON format that can be more suitable for use in dApps. When building
+the contract use ``--schema-json-out=DIR`` to output the schema for each
+contract in the module to a JSON file inside the directory ``DIR``. The
+directory must exist.
+
+.. code-block:: console
+
+   $cargo concordium build --schema-json-out="/some/path"
+
+Both ``--schema-out`` and ``--schema-json-out`` can be used at the same time and
+schemas in both formats will be output.
+
+.. code-block:: console
+
+   $cargo concordium build --schema-out="/some/path/schema.bin" --schema-json-out="/some/path"
+
+Converting a binary schema to JSON
+----------------------------------
+
+To convert an existing binary schema (obtained via ``--schema-out``) use the
+``cargo concordium schema-json`` subcommand, e.g.,
+
+.. code-block:: console
+
+   $cargo concordium schema-json --schema schema/schema.bin --out="/some/path"
+
+Alternatively, a schema in JSON can be extracted from an embedded schema by using
```

Comment:
We could also mention how this module was obtained so that it has an embedded schema in it.

```suggestion
Alternatively, a schema in JSON can be extracted from an embedded schema in a module by using

Note: Such a module can be obtained with the command ``cargo concordium build -e --out ./module.wasm.v1``.
```

Reply:
Good point.